$MirOS: ports/devel/libusb/patches/patch-configure_in,v 1.1.7.1 2005/03/18 15:44:55 tg Exp $
--- configure.in.orig	Mon Feb 14 20:21:46 2005
+++ configure.in	Thu Dec 29 21:08:58 2005
@@ -98,16 +98,16 @@ case $host in
     AC_DEFINE(DARWIN_API, 0)
     LINUX_API=1
     os_support=linux
-    AC_MSG_RESULT(Linux)
+    AC_MSG_RESULT(GNU/Linux)
     OSLIBS=""
     ;;
-  *-freebsd*|*-openbsd*|*-netbsd*)
+  *-freebsd*|*-openbsd*|*-netbsd*|*-mirbsd*)
     AC_DEFINE(BSD_API, 1)
     AC_DEFINE(LINUX_API, 0)
     AC_DEFINE(DARWIN_API, 0)
     BSD_API=1
     os_support=bsd
-    AC_MSG_RESULT(FreeBSD, OpenBSD and/or NetBSD)
+    AC_MSG_RESULT(FreeBSD, MirBSD, OpenBSD and/or NetBSD)
     OSLIBS=""
     ;;
   *-darwin*)
@@ -191,11 +191,9 @@ AC_CHECK_FUNCS(memmove)
 
 if test "$os_support" = "bsd"; then
   AC_MSG_CHECKING(if dev/usb/usb.h uses new naming convention)
-  AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <dev/usb/usb.h>]], [[int main(void)
-     {
+  AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <dev/usb/usb.h>]], [[
        int a = ((struct usb_ctl_request *)0L)->ucr_addr;
-       return 0;
-     }]])],[AC_MSG_RESULT(yes)],[AC_MSG_RESULT(no)
+     ]])],[AC_MSG_RESULT(yes)],[AC_MSG_RESULT(no)
     AC_DEFINE_UNQUOTED(HAVE_OLD_DEV_USB_USB_H)])
 fi
 
